### PR TITLE
DRAFT - RDAPI-18479 - API Manager request rate limiter

### PR DESCRIPTION
**Pull request by credmond-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by credmond-axway** [comment]
@JuanAxway @kmercierwhite Please review.

Preview can be seen [here](https://deploy-preview-1572--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_config/#configure-the-api-manager-request-rate-limiter)

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md [hunk 1]
```diff
@@ -102,3 +97,19 @@ The following JVM setting is also required when importing the API in API Manage
...
 
 For details on configuring API Manager for a large amount of APIs and data in a multi-datacenter environment, see [Configure API Management in multiple datacenters](/docs/apimgmt_multi_dc/).
+
+## Configure the API Manager request rate limiter
+
+The request rate limiter monitors the number of requests an individual user makes to API Manager during an active session. If the number of user requests in an individual session exceeds the configured boundaries, that user will be automatically logged out and will need to login again to continue using API Manager.
+
+To configure the request rate limiter, perform the following steps in Policy Studio:
+
+1. Open the configuration of your API Manager-enabled API Gateway instance. For example, select **File > New Project from an API Gateway instance**.
```
Comment (kmercierwhite) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md:107]: All the other similar items in this page assumes that the user has an API Manager project setup. This step can be omitted.

**Comment by kmercierwhite** [comment]
Just the minor change. I'm happy with the rest. Thanks Cian!

**Comment by JuanAxway** [comment]
@kmercierwhite, I have sent an email with my suggestion, I can't replicate it here because of character errors.

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md (hunk 1 above)
Comment (credmond-axway) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md]: Thanks Kelly. I've removed that step.

**Review by JuanAxway — APPROVED** [review]

**Review by kmercierwhite — APPROVED** [review]

**Comment by credmond-axway** [comment]
Closing this PR as this feature has been deferred to Mar21. I will create a new PR once the March release cycle starts

**Pull request closed without merging**